### ISSUE TITLE
Dataset color

### ISSUE THREAD

**Issue by hmihail** [ConsoleTVs/Charts#19]
Is there a way to set the dataset color? I have a chart (line) with multiple datasets, and all of them appear gray. Is there a way to set the color of each dataset? I am using chartjs.

Thanks!

**Comment by artifex-media** [comment]
Cannot find it either, any news?

**Comment by chaoswey** [comment]
Absolutely—after you call dataset(...) you can chain the helper methods that ConsoleTVs/Charts exposes for Chart.js options. For colors, the relevant helpers live
  in ConsoleTVs\Charts\Features\Chartjs\Dataset. A basic example:
```
  $chart->dataset('Dataset A', 'line', [5, 10, 4])
      ->color('#2563eb') // sets Chart.js borderColor
      ->backgroundColor('rgba(37,99,235,0.2)') // sets backgroundColor (fill)
      ->borderWidth(2);
```
  If you have multiple datasets, repeat the chain for each one with different values (strings for single colors, arrays if you need per-point colors). When the Blade
  view renders, the generated dataset JSON already carries those colors, so Chart.js uses them directly—no extra changes needed on the front end.
